lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2315,6 +2335,18 @@ function EDITOR:AC_Check( notimer )
 
 		local word, _ = self:AC_GetCurrentWord()
 
+		-- PickColorFind
```

Comment:
I don't like this being scattered everywhere. If just searching for `PickColor` in the codebase isn't useful, then having to do this is a sign that the code should be organized better.